src/flows/components/AddButtons.tsx

```diff
@@ -80,15 +82,27 @@ const AddButtons: FC<Props> = ({index, onInsert}) => {
     return acc
   }, {})
 
+  const pipeIds = flow.data.allIDs
+  const firstQueryIndex = pipeIds.findIndex(
+    id =>
+      flow.data.byID[id].type === 'queryBuilder' ||
```

Comment:
can please key off the family here instead of the type in order to allow for more types to be added?

Reply:
done and done!